python/lsst/ts/salobj/base_csc.py

```diff
@@ -177,14 +184,21 @@ class BaseCsc(Controller):
     * Implement :ref:`simulation mode<simulation_mode>`, if practical.
       If your CSC talks to hardware then this is especially important.
 
+    .. _running_a_csc:
+
+    **Running a CSC**
+
     To run your CSC call the `main` method or equivalent code.
     For an example see ``bin.src/run_test_csc.py``. If you wish to
     provide additional command line arguments then it is probably simplest
     to copy the contents of `main` into your script and adapt it as required.
 
+    In unit tests, wait for `start_task` to be done, or for the initial
+    summaryState event, before expecting the CSC to be responsive.
+
     .. _simulation_mode:
 
-    Simulation Mode:
+    **Simulation Mode**
 
     CSCs should support a simulation mode it practical; this is especially
```

Comment:
Typo, "CSCs should support a simulation mode if practical; this is especially important if the CSC talks to the hardware"